develop/develop/settings.py

```diff
@@ -182,7 +182,10 @@
 # Stripe Settings
 STRIPE_TEST_SECRET_KEY = os.getenv("STRIPE_TEST_SECRET_KEY")
 STRIPE_TEST_PUBLIC_KEY = os.getenv("STRIPE_TEST_PUBLIC_KEY")
+STRIPE_LIVE_SECRET_KEY = os.getenv("STRIPE_LIVE_SECRET_KEY")
+STRIPE_LIVE_PUBLIC_KEY = os.getenv("STRIPE_LIVE_PUBLIC_KEY")
 STRIPE_LIVE_MODE = False
+STRIPE_API_KEY = STRIPE_TEST_SECRET_KEY if VENDOR_STATE == "DEBUG" else STRIPE_LIVE_SECRET_KEY
```

Comment:
I prefer to have one variable instead of doubles to keep the same as with AuthorizeNet.


Change the to `STRIPE_SECRET_KEY` and `STRIPE_PUBLIC_KEY`

Reply:
There's no sandbox or test server like authorize.net. Everything hits the same server, but its distinguished if you're testing based on if its a test key or live key. Both need need to be there